### PR TITLE
[Lettuce][|UI] Implement messages thread view 

### PR DESCRIPTION
**Pull request by dominika-kokowicz**
Added assets, added message view and placeholder for viewModel and popup

### CODE REVIEW

File: App/Seller/Sources/Seller/UI/MessagesView/MyMessagesView.swift [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+
+
+protocol SizingFrameType {
+    
+    static var instance: SizingFrameType { get }
+    
+    var size: CGSize { get }
+}
+
+#if os(macOS)
```
Comment (sloik) [inline App/Seller/Sources/Seller/UI/MessagesView/MyMessagesView.swift:17]: Do you have `main` merged in as I can see this change already exists.

File: App/Seller/Sources/Seller/UI/MessagesView/MyMessagesView.swift [hunk 2]
```diff
@@ -0,0 +1,194 @@
+// system
+import SwiftUI
+
+// Internal
+
+// 3rd
+import OptionalAPI
+
+
+protocol SizingFrameType {
```
Comment (sloik) [inline App/Seller/Sources/Seller/UI/MessagesView/MyMessagesView.swift:10]: Maybe this should be moved to `Utilities`? :)

File: App/Seller/Sources/Seller/UI/MessagesView/MyMessagesView.swift [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+    }
+    
+    public var size: CGSize {
+        bounds.size
+    }
+}
+
+#endif
+
+struct MessagesView: View {
```
Comment (sloik) [inline App/Seller/Sources/Seller/UI/MessagesView/MyMessagesView.swift:48]: Would be nice to extract those views to files :)

**Review by sloik — APPROVED** [review]
🏆

File: App/Seller/Sources/Seller/UI/MessagesView/MyMessagesView.swift (hunk 1 above)
Comment (dominika-kokowicz) [reply App/Seller/Sources/Seller/UI/MessagesView/MyMessagesView.swift:17]: Weird, main should have been (and imo was) merged :D

File: App/Seller/Sources/Seller/UI/MessagesView/MyMessagesView.swift [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+}
+
+#if os(macOS)
+import AppKit
+public typealias PlatformScreen = NSScreen
+
+extension NSScreen: SizingFrameType {
+    
+    static var instance: SizingFrameType {
+        NSScreen.main!
```
Comment (Dentro2000) [inline App/Seller/Sources/Seller/UI/MessagesView/MyMessagesView.swift:24]: is this force unwrap safe?

File: App/Seller/Sources/Seller/UI/MessagesView/MyMessagesView.swift [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+
+struct MessagesView: View {
+    @ObservedObject private var viewModel = MyMessagesViewModel()
+    
+    var body: some View {
+        VStack(spacing: 0) {
+            NavigationView(viewModel: viewModel)
+                .frame(height: 77)
+            ScrollView {
+                VStack(spacing: 17) {
```
Comment (Dentro2000) [inline App/Seller/Sources/Seller/UI/MessagesView/MyMessagesView.swift:56]: why 77 and 17? What do you think about making those relative to something, like: let navigationViewHeigh = screenHeigh/2? Is it possible?
I see below that some values are reused. What do you think about extracting those to variables and reuse?

Comment (dominika-kokowicz) [reply App/Seller/Sources/Seller/UI/MessagesView/MyMessagesView.swift:56]: It is possible with geometry reader, although at this point it is hard to specify the best approach imo, as those relative values should be adjusted to size classes and dimensions usually.

Magic numbers usually appear when they are not repeating in a context in this scenario, I am planning to add task for creating Theme for fonts, colors, and some reusable paddings.

File: App/Seller/Sources/Seller/UI/MessagesView/MyMessagesView.swift (hunk 1 above)
Comment (dominika-kokowicz) [reply App/Seller/Sources/Seller/UI/MessagesView/MyMessagesView.swift:17]: Checked again, rebased xD

File: App/Seller/Sources/Seller/UI/MessagesView/MyMessagesView.swift (hunk 2 above)
Comment (dominika-kokowicz) [reply App/Seller/Sources/Seller/UI/MessagesView/MyMessagesView.swift]: Moved

File: App/Seller/Sources/Seller/UI/MessagesView/MyMessagesView.swift (hunk 3 above)
Comment (dominika-kokowicz) [reply App/Seller/Sources/Seller/UI/MessagesView/MyMessagesView.swift:9]: Extracted :D

File: App/Seller/Sources/Seller/UI/MessagesView/MyMessagesView.swift (hunk 5 above)
Comment (Dentro2000) [reply App/Seller/Sources/Seller/UI/MessagesView/MyMessagesView.swift:17]: 👍

**Pull request merged**